docs/UserGuide.md

```diff
@@ -70,6 +70,9 @@ Adds an animal to the address book.
 Format: `add n/NAME i/ID g/SEX s/SPECIES b/BREED db/DOB da/DOA`
 
 * ID must be a unique 4 digit number.
+* SEX must be either `male` or `female`.
+* DOB and DOA must be in the format `YYYYMMDD`.
+* Species and breed can only be alphabetic with only a single space between words.
```

Comment:
For this, might be good to specify how many words can it contain! I think our implementation only allows 2 words max with a space between, but this line might imply unlimited number of words as long as there are spaces

Reply:
Yep you are right!